pom.xml

```diff
@@ -150,6 +150,7 @@ THE SOFTWARE.
             <groupId>org.mockito</groupId>
             <artifactId>mockito-inline</artifactId>
             <scope>test</scope>
+            <version>5.2.0</version>
```

Comment:
this is not correct - `mockito-inline` is no more and should be just replaced with `mockito` 

When bumping parent always be sure to heck all the release notes :)
https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.60